<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-PDMEH2U" name="testNGTestcases.TestNGGroups" tests="5" failures="1" timestamp="2022-09-23T23:11:03 IST" time="0.024" errors="0">
  <testcase name="validatingImage" time="0.000" classname="testNGTestcases.TestNGGroups">
    <skipped/>
  </testcase> <!-- validatingImage -->
  <system-out/>
  <testcase name="validatingTitle" time="0.005" classname="testNGTestcases.TestNGGroups"/>
  <system-out/>
  <testcase name="validatingMessage" time="0.002" classname="testNGTestcases.TestNGGroups"/>
  <system-out/>
  <testcase name="doUserRegister" time="0.015" classname="testNGTestcases.TestNGGroups">
    <failure type="java.lang.AssertionError" message="Not able to register">
      <![CDATA[java.lang.AssertionError: Not able to register
at org.testng.Assert.fail(Assert.java:97)
at testNGTestcases.TestNGGroups.doUserRegister(TestNGGroups.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doUserRegister -->
  <system-out/>
  <testcase name="doLogin" time="0.002" classname="testNGTestcases.TestNGGroups"/>
  <system-out/>
</testsuite> <!-- testNGTestcases.TestNGGroups -->
